settings_vial/exceptions.py

```diff
@@ -0,0 +1,16 @@
+class NotCallable(Warning):
```

Comment:
minor (naming): for clarity, I think these exceptions should suffixed with `Exception` or `Error`.

Reply:
Maybe `Warning` then?